main.tf

```diff
@@ -41,6 +43,7 @@ module "pubsub_topic" {
   version    = "~> 1.0"
   topic      = var.topic_name
   project_id = var.project_id
+  enable     = var.scheduler_job == null ? true : false
```

Comment:
```suggestion
  create_topic     = var.scheduler_job == null ? true : false
```

Reply:
+ change line 60 to `resource = var.scheduler_job == null ? module.pubsub_topic.topic : var.topic_name`

Reply:
With those 2 changes it should do it 👍